.github/workflows/test.yaml

```diff
@@ -30,6 +30,11 @@ jobs:
     steps:
     - name: Checkout Repo
       uses: actions/checkout@v3.3.0
+
+    - name: Install noisepy-seis-io
```

Comment:
We shouldn't need this. If we do it's indicative of a problem with having it as a dependency.